spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
```

Comment:
- [ ] Also for best practices and as required for the project, kindly ensure that both your `users_controller_spec.rb` and `posts_controller_spec.rb` spec files are in the `spec/requests/` directory. This means you should edit/change this directory name `spec/controllers/` to ``spec/requests/``